### ISSUE TITLE
Cloak / Cape

### ISSUE THREAD

**Issue by iAskel** [iChun/BackTools#32]
I can use Back Tools and cloak (with Optifine and CustomSkinLoader mod)? If i enable cloak tool doesnt show, how i can enable it?

**Comment by iChun** [comment]
By default the mod disables rendering if the player has a cape because it clips with the cape. I guess I could add a config for this in later versions of the mod

**Comment by iAskel** [comment]
Yes I know that the sword hanging on his back will fall through the texture of the cloak, but I think in this game it is not so important in terms of realism, but such a function would not hurt. One more question, is it possible to do in Minecraft to make a sword hanging not on the back and let's say on the side (right or left), on the belt as is usually the case in rpg games? With the mod First-person Model it would look especially cool, although on the back it looks decent.  It looks good in third-person, too.
